### ISSUE TITLE
Character Login Bug

### ISSUE THREAD

**Issue by RaginnFlame** [zajako/LineageHD#453]
4:51 pm Pacific

This happens quite often, but this is my first time reporting it. when I log into a character, everything  froze except my character can still move.  The inventory is messed up, and all the hot key box are reset.  I cannot restart the character,  must close the game and restart, and upon first time restarting it, it will have to log me out once and I have to restart again to log in. 

Wondering if this just happened to me or everyone elses have the same problem.

**Comment by RaginnFlame** [comment]
![Uploading Screenshot_20230607_165621_Gallery.jpg…]()

**Comment by RaginnFlame** [comment]
Now I cant even start the game. 
![Uploading 20230607_170335.jpg…]()

Only thing i can see from the error code is 101 everything else is random text.

**Comment by zajako** [comment]
This is no longer happening correct?

**Comment by RaginnFlame** [comment]
> This is no longer happening correct?

Correct, it havent happened in a while, but character black screen happened twice I think, but the inventory havent get messed up.

**Comment by Fawx907** [comment]
if this is about or part of the inventory scrambling after a black screen login then that happened to me 2 days ago.

**Comment by RaginnFlame** [comment]
it happened again. On WinDBreaker. I think last night there were two black screen crashes, I had to force quit and come back. 
When I open Windbreaker this morning, all her item is messed up and all hot key is removed.
